core/wiki/macros/tag-picker.tid

```diff
@@ -68,16 +69,24 @@ tags: $:/tags/Macro
 <$set name="userInput" value={{{ [<storeTitle>get[text]] }}}>
 <$list filter="[<userInput>minlength{$:/config/Tags/MinLength}limit[1]]" emptyMessage="""<div class="tc-search-results">{{$:/language/Search/Search/TooShort}}</div>""" variable="listItem">
 <$list filter=<<filter1>> variable="tag">
-<$list filter="[<newTagNameTiddler>get[text]match<tag>]" emptyMessage="""<<tag-button>>""">
+<$set name="state-value" filter="[<q-state>get[text]]">
```

Comment:
We should be able to make do with one less set widget, I was being conservative while trying to get this working. Will optimize later.